pyproject.toml

```diff
@@ -29,10 +23,10 @@ requires = ["poetry_core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
-python = "^3.6"
-flake8 = ">=3.8.4,<5"
+python = "^3.7"
+flake8 = ">=3.8.4,<6"
```

Comment:
Can you add a note about this bump to the CHANGELOG? I'll merge this and publish the version after that's updated. Thank you!

Reply:
Done.